module/VuFindConsole/tests/unit-tests/src/VuFindTest/Command/Util/SwitchDbHashCommandTest.php

```diff
@@ -285,4 +284,18 @@ public function testSuccessWithUser()
         $this->assertEquals(null, $user['cat_password']);
         $this->assertEquals('mypassword', $this->decode($user['cat_pass_enc']));
     }
+
+    /**
+     * Get path to local config.ini
+     *
+     * @return string
+     */
+    protected function getLocalConfigIniPath(): string
+    {
+        $localDirs = defined('LOCAL_OVERRIDE_DIR')
```

Comment:
Maybe this would be better done in a setup method so we don't keep rebuilding the object over and over... and we could theoretically use the factory.